ONSdigital/eq-author-app
# Update to handle qCodes for other options on radio and checkbox

### Context of PR ###

Ability to add qCodes for `other` radio and checkbox labels.

### How to review ###

1. Create a radio and checkbox other option with a label.
2. Go into qCodes. 
3. Check that you can now add and update the qCodes for the other labels for both radio and checkbox.
4. Check that the publisher schema also has the new qCode under `detail_answer` using `http://localhost:9000/publish/`

What to do after everything is green
    *   Bring this branch up-to-date with Origin/Master
    *   If there are a lot of commits or some are not helpful to read, squash them down
    *   Click the Merge button on this pull request
    *   Does this change mean the Capability examples questionnaire should be updated?
    *   Move the Jira ticket for this task into the next stage of the process
